conans/assets/templates/new_v2_autotools.py

```diff
@@ -38,6 +37,7 @@ def layout(self):
 
         def generate(self):
             at_toolchain = AutotoolsToolchain(self)
+            at_toolchain.configure_args = at_toolchain.shared_flags()
```

Comment:
So now the management of shared flag is not by default?

Reply:
I was thinking that maybe we can make it by default again as you have the possibility now to override all defaults...